client/asset/ltc/ltc.go

```diff
@@ -0,0 +1,112 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package ltc
+
+import (
+	"fmt"
+	"os"
+
+	"decred.org/dcrdex/client/asset"
+	clientbtc "decred.org/dcrdex/client/asset/btc"
+	"decred.org/dcrdex/dex"
+	dexbtc "decred.org/dcrdex/dex/btc"
+	"decred.org/dcrdex/server/asset/btc"
+	chaincfgbtc "github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/ltcsuite/ltcd/chaincfg"
+)
+
+const (
+	BipID = 2
+	// The default fee is passed to the user as part of the asset.WalletInfo
+	// structure.
+	defaultWithdrawalFee = 2
```

Comment:
I think the current network minimum is `1`. I'm okay with `2` if it's based on network statistics, but otherwise a default of `1` is preferred. Eventually, it would be nice if we offered a way to control this.